CoreScriptsRoot/StarterScript.lua

```diff
@@ -66,3 +66,30 @@ if useVRKeyboard then
 	require(RobloxGui.Modules.VR.VirtualKeyboard)
 end
 
+
+-- Boot up the VR App Shell
+if UserSettings().GameSettings:InStudioMode() then
```

Comment:
If it would be preferable, we could refactor this following codeblock into its own module. That way we don't have complicated logic in the StarterScript